docs/checks/invalidTagsCheck.md

```diff
@@ -78,16 +78,25 @@ In [Atlas](https://github.com/osmlab/atlas), OSM elements are represented as Edg
 
 Due to the general nature of this check, most features are valid for this check.
-The only validation requirement is that a feature of the same type and OSM ID has not been flagged before. 
-Many other checks only look at the OSM ID when checking if a feature is flagged. Because this check deals 
+The only validation requirement is that a feature of the same type and OSM ID has not been flagged before.
+Many other checks only look at the OSM ID when checking if a feature is flagged. Because this check deals
 with multiple types of features, and ids are not unique across types, this check has to check the type and id.
 
 Once a feature has passed validation it is checked against the filters that are set in the constructor of the check, based on the config values mentioned in the check description.
 For each filter the feature is checked that it is of a type given in the first part of the filter
-and passes the filter. If both these things are true then the feature is flagged. 
+and passes the filter. If both these things are true then the feature is flagged.
 
-For each filter a feature passes an instruction is added to the flag. The tag keys from the TaggableFilter and the 
-RegexTaggableFilter are pulled out and added to the instruction to give guidance on where the problem is. 
+For each filter a feature passes an instruction is added to the flag. The tag keys from the TaggableFilter and the
+RegexTaggableFilter are pulled out and added to the instruction to give guidance on where the problem is.
 
-To learn more about the code, please look at the comments in the source code for the check:  
+The `fallback` check first checks to see if the tag has been fully checked by another check, then it checks to see if the tag is in OpenStreetMap Wiki Data (must be downloaded separately, see [get_wikidata.py](../../scripts/get_wikidata.py)).
+If the tag does _not_ exist in OSM WikiData AND the key does exist and specifies that all values are `well-known`, an instruction is generated.
+If the tag does _not_ exist in OSM WikiData AND the key does exist AND the key specifies a value validation regex AND the value fails the regex, an instruction is generated.
+If the tag does exist in the OSM WikiData AND the key has been marked as unwanted, an instruction is generated.
+If the tag does _not_ exist in OSM WikiData AND the key does not exist THEN the TagInfo database is queried. If the uses of the tag is low (less than the `tag.usage.min` variable, or `100` if the variable is unset), an instruction for an editor is generated, otherwise an instruction for a wiki editor is generated.
+
+For each filter a feature passes an instruction is added to the flag. The tag keys from the TaggableFilter
+are pulled out and added to the instruction to give guidance on where the problem is.
+
+To learn more about the code, please look at the comments in the source code for the check:
```

Comment:
Looks like this part was added by mistake here?

Reply:
It was left by mistake, yes. I was originally putting the code in the `InvalidTagsCheck` file, and later decided to move it out.